open_spiel/game_transforms/coop_to_1p.h

```diff
@@ -163,6 +163,8 @@ class CoopTo1pGame : public Game {
   int NumPlayers() const override { return 1; }
   std::vector<int> ObservationTensorShape() const override;
   int MaxGameLength() const override;
+  // TODO() make a tighter bound.
```

Comment:
Can you re-format all the TODOs as the following (notice no parentheses): 

TODO: verify whether this bound is tight and/or tighten it.